src/metabase/driver/sql_jdbc/actions.clj

```diff
@@ -0,0 +1,47 @@
+(ns metabase.driver.sql-jdbc.actions
+  (:require
+   [clojure.java.jdbc :as jdbc]
+   [honeysql.format :as hformat]
+   [metabase.actions :as actions]
+   [metabase.driver.sql-jdbc.connection :as sql-jdbc.conn]
+   [metabase.driver.sql.query-processor :as sql.qp]
+   [metabase.util.i18n :as i18n]
+   [metabase.query-processor.store :as qp.store]
+   [metabase.util :as u]))
+
+(defmethod actions/row-action! [:delete :sql-jdbc]
+  ;; "Often condition is a map of primary-key(s) => value(s)."
+  [_action driver {database-id :database :as query}]
+  (let [connection-spec        (sql-jdbc.conn/db->pooled-connection-spec database-id)
+        _ (require '[metabase.test :as mt]) ;;HACK: using mt here is not good
```

Comment:
I have a feeling there is an easy fix for this, but I looked for a while and couldn't find one.